test/integrations/component.test.ts

```diff
@@ -43,6 +43,7 @@ command
   .option('-id, --id <string>', 'Enter unique "Id" of the test case you want to run')
   .parse();
 
+jest.setTimeout(30000000);
```

Comment:
is it required?

Reply:
My bad! This was done for debugging purposes

Reply:
Will revert

Reply:
Removed this. Resolving the comment